hikari/impl/stateful_cache.py

```diff
@@ -969,7 +999,9 @@ def get_guild(self, guild_id: snowflake.Snowflake, /) -> typing.Optional[guilds.
 
     def get_guilds_view(self) -> cache.CacheView[snowflake.Snowflake, guilds.GatewayGuild]:
         results = {
-            sf: guild_record.guild for sf, guild_record in tuple(self._guild_entries.items()) if guild_record.guild
+            sf: guild_record.guild
+            for sf, guild_record in _copy_mapping(self._guild_entries).items()
```

Comment:
When would it be falsy? Comments for context around stuff like this just to explain would make life 100x easier for anyone else who may need to work with this code in the future (just remember not everyone is as tuned in to how this all works as you are! Descriptions are really useful in dense classes of code!)